src/tpm2tss_engine.c

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright 2020 Xaptum, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <openssl/ssl.h>
+#include <openssl/ui.h>
+#include <openssl/engine.h>
+#include <stdio.h>
+#include <string.h>
+#include "log.h"
+#include "tpm2tss_engine.h"
+
+static ENGINE *
+enftun_tpm2tss_engine_init()
```

Comment:
Some of the static functions include the `enftun_tls_tpm` prefix.  Others, like the `ui_` functions do not.   Pick one approach and use it consistently.

Something like 
```
static ENGINE*
tpm_engine_init(...)
```
would probably be fine.

Reply:
`enftun_tls_tpm_*` is used across the board now.